sunpy/net/dataretriever/sources/lyra.py

```diff
@@ -45,7 +45,8 @@ def _get_url_for_date(self, date, **kwargs):
             The URL for the corresponding date.
         """
 
-        filename = "lyra_{0:%Y%m%d-}000000_lev{1:d}_std.fits".format(date, kwargs.get('level', 2))
+        filename = "lyra_{0}000000_lev{1:d}_std.fits".format(
+            date.strftime('%Y%m%d-'), kwargs.get('level', 2))
```

Comment:
what does the `-` do?

Reply:
It was there initially. So I left it untouched. It should print something like `20180101-`.

Reply:
I would have thought that would be more at home in the `"lyra...` string rather than the argument to `strftime`.

Reply:
Oh. Yeah. That is true.